docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -202,10 +225,7 @@ If everything works fine, the log output should look like this:
 
 ```bash
 $ teleport-slack start
-INFO   Starting Teleport Access Slack (=teleport.plugin.version=).1-0-slack/main.go:145
-INFO   Starting a request watcher... slack/main.go:330
-INFO   Starting insecure HTTP server on 0.0.0.0:8081 utils/http.go:64
-INFO   Watcher connected slack/main.go:298
+INFO   Starting Teleport Access Slack (=teleport.plugin.version=)-slack/main.go:145
```

Comment:
Just want to verify this change:

1. I still see: https://github.com/gravitational/teleport-plugins/blob/8bb9e0846530a274783c04fdf3442e9f9bfbebff/lib/http.go#L243
2. Also: https://github.com/gravitational/teleport-plugins/blob/8bb9e0846530a274783c04fdf3442e9f9bfbebff/access/service_job.go#L64
3. I also don't see the `Starting a request watcher...` message: https://github.com/gravitational/teleport-plugins/blob/8bb9e0846530a274783c04fdf3442e9f9bfbebff/access/slack/main.go#L30 - did a GREP on the codebase and I'm not sure it's been in there for several months? 

I have been unable to reproduce these messages (except for the last one): https://github.com/gravitational/teleport-plugins/issues/230

These also need to be updated: https://github.com/gravitational/teleport-plugins/blob/master/access/slack/INSTALL.md#test-run

Reply:
This has indeed been changed to `INFO   Starting Teleport Access Slack Plugin 6.2.0: slack/app.go:72` the other messages no longer appear.